docs/how-to-guides.md

```diff
@@ -32,6 +32,199 @@ Everything about test data generation and its adjustable parameters can
 be found in the [Generating a Test Dataset with niceML](
 generate-data.md#step-3-customizing-data-generation-optional)
 
+## How to write my own pipeline
```

Comment:
For me, this section feels like a whole tutorial rather than a how to. I suggest to move the section to the Tutorials Part of the Documentation.